SETUP_LOCAL_CLUSTER.md

```diff
@@ -0,0 +1,70 @@
+# Introduction
+
+For both HOPR app and core protocol developers, being able to spin a fully interconnected HOPR network is critical.
```

Comment:
Who is the recipient of this text?

Reply:
`app` and `core protocol` developers, I believe.

Reply:
Might sound strange, but isn't this a little bit too direct? It feels like someone explaining me how to do my job

Reply:
Fair take, I can soften it up, one sec.

Reply:
Improved this in 90d76bf, please feel free to provide some feedback there.